src/app/qgsoptions.cpp

```diff
@@ -144,7 +144,11 @@ QgsOptions::QgsOptions( QWidget *parent, Qt::WindowFlags fl, const QList<QgsOpti
   QStringList themes = QgsApplication::uiThemes().keys();
   cmbUITheme->addItems( themes );
 
+#if QT_VERSION < QT_VERSION_CHECK( 5, 12, 0 )
```

Comment:
From https://bugreports.qt.io/browse/QTBUG-69204 this is still an issue in 5.12.0. 
```suggestion
#if QT_VERSION >= QT_VERSION_CHECK(5, 11, 0)
```